src/firejail/seccomp.c

```diff
@@ -157,6 +161,11 @@ void filter_debug(void) {
 			printf("  BLACKLIST %d %s\n", *nr, syscall_find_nr(*nr));
 			i += 2;
 		}
+		else if (*ptr	== 0x15 && *(ptr +14) == 0x5 && *(ptr + 15) == 0) {
```

Comment:
maybe this function could be rewritten using sock_filter struct pointer instead hmm would also take care of the endian stuff